tests/routes/test_document_families.py

```diff
@@ -323,6 +325,65 @@ def test_update_document__works_on_import_id(
     assert doc.cdn_object == "folder/file"
 
 
+@pytest.mark.parametrize(
+    "import_id",
+    [
+        "CCLW.executive.1.2",
+        "UNFCCC.non-party.2.2",
+    ],
+)
+def test_update_document__works_on_new_language(
```

Comment:
I think we also need tests for setting languages on a document that already has languages set, both where the updates overlap with the existing ones, and where they don't. We need to confirm that we end up with the combination of existing & updates each time and that no errors are thrown.

Reply:
Surely the initial documents that are loaded into the database via the following code on line 343 would load have the language Algeria already? 

python```
setup_with_multiple_docs(
        test_db, mocker, doc_data=TWO_DFC_ROW_DIFFERENT_ORG, event_data=TWO_EVENT_ROWS
)
```

https://github.com/climatepolicyradar/navigator-backend/blob/feature/update-document-language-via-endpoint/tests/routes/document_helpers.py#L26

My understanding is that we go: 
1. Algeria 
2. Delete data
3. Load English and French